boards/xtensa/intel_adsp_cavs15/tools/adsplog.py

```diff
@@ -40,31 +40,39 @@
 WIN_SIZE = 0x20000
 LOG_OFFSET = WIN_OFFSET + WIN_IDX * WIN_SIZE
 
-# List of known ADSP devices by their PCI IDs
-DEVICES = ["8086:5a98"]
-
 mem = None
+pcid="/sys/bus/pci/devices"
+
+for fname in os.listdir(pcid):
+    pfile = pcid + "/" + fname + "/class"
+    if not os.path.exists(pfile):
+        continue
```

Comment:
According to https://wiki.osdev.org/PCI#PCI_Device_Structure the class is mandatory so you can drop these two lines (there is an "Unclassified", non-empty class code)

Reply:
Ok, will do